Makefile

```diff
@@ -26,20 +27,23 @@ integration: start-api
 validate-version:
 	cat vinyldns/version.go | grep 'var Version = "$(VERSION)"'
 
-start-api:
-	if [ ! -d "$(GOPATH)/src/$(VINYLDNS_REPO)-$(VINYLDNS_VERSION)" ]; then \
-		echo "$(VINYLDNS_REPO)-$(VINYLDNS_VERSION) not found in your GOPATH (necessary for acceptance tests), getting..."; \
+clonevinyl:
```

Comment:
Everything looks good to me expect this minor typo change.

Kindly update clonevinyl to clone-vinyl.

Reply:
@Jay07GIT corrected